examples/plot_mne_epoch_masking.py

```diff
@@ -41,6 +41,10 @@
 # results.
 
 # create the input array for Comodulogram.fit
+
+# low_sig, high_sig = raw[8, :][0], raw[10, :][0]
+# mask = MaskIterator(events, tmin, tmax, raw.n_times, raw.info['sfreq'])
```

Comment:
Is this equivalent to the lines below? I think this might be somewhat more general in that case because it would work also for `stc` objects not just `raw` ...

Reply:
This is equivalent indeed, nice !